src/main/app/case/CaseApi.ts

```diff
@@ -121,31 +120,31 @@ export class CaseApi {
     }
   }
 
-  // private async sendEvent(caseId: string, data: Partial<CaseData>, eventName: string): Promise<CaseWithId> {
-  //   try {
-  //     const tokenResponse = await this.axios.get<CcdTokenResponse>(`/cases/${caseId}/event-triggers/${eventName}`);
-  //     const token = tokenResponse.data.token;
-  //     const event = { id: eventName };
-
-  //     const response: AxiosResponse<CcdV2Response> = await this.axios.post(`/cases/${caseId}/events`, {
-  //       event,
-  //       data,
-  //       event_token: token,
-  //     });
-  //     //...fromApiFormat(response.data.data)
-  //     return { id: response.data.id, state: response.data.state, ...response.data.data };
-  //   } catch (err) {
-  //     this.logError(err);
-  //     throw new Error('Case could not be updated.');
-  //   }
-  // }
+  private async sendEvent(caseId: string, data: Partial<CaseData>, eventName: string): Promise<CaseWithId> {
+    try {
+      const tokenResponse = await this.axios.get<CcdTokenResponse>(`/cases/${caseId}/event-triggers/${eventName}`);
+      const token = tokenResponse.data.token;
+      const event = { id: eventName };
+
+      const response: AxiosResponse<CcdV2Response> = await this.axios.post(`/cases/${caseId}/events`, {
+        event,
+        data,
+        event_token: token,
+      });
+      //...fromApiFormat(response.data.data)
+      return { id: response.data.id, state: response.data.state, ...response.data.data };
+    } catch (err) {
+      this.logError(err);
+      throw new Error('Case could not be updated.');
+    }
+  }
 
   public async triggerEvent(caseId: string, userData: Partial<Case>, eventName: string): Promise<CaseWithId> {
     //const data = toApiFormat(userData);
     //const data = userData;
     // return this.sendEvent(caseId, data, eventName);
     console.log(eventName);
-    return { id: caseId, state: State.successAuthentication, serviceType: '', ...userData };
+    return this.sendEvent(caseId, userData, eventName);
```

Comment:
Seems we are not doing any logic here in this method. Can we remove this method and directly call sendEvent() method instead?